main.js

```diff
@@ -1,10 +1,13 @@
 const fs = require('fs');
 
-const textExample = fs.readFileSync('testData/dog.css').toString();
+const textExample1 = fs.readFileSync('testData/cat.html', 'utf-8');
+const textExample2 = fs.readFileSync('testData/dog.css', 'utf-8');
+const textExample3 = fs.readFileSync('testData/world.txt', 'utf-8');
+const addText = new Array(textExample1, textExample2, textExample3);
```

Comment:
Please rename `addText` to something like `textsForAnalysis`

Reply:
Changes made